src/Sarif.Multitool/FileWorkItemsOptions.cs

```diff
@@ -9,31 +9,22 @@ namespace Microsoft.CodeAnalysis.Sarif.Multitool
 {
     [Verb("file-work-items", HelpText = "Send SARIF results to a work item tracking system such as GitHub or Azure DevOps")]
     public class FileWorkItemsOptions : SingleFileOptionsBase
-    {
+    { 
         [Option(
             "project-uri",
             HelpText = "The absolute URI of the project in which the work items are to be filed, for example https://dev.azure.com/{org}/{project} or https://github.com/{org}/{project}.",
             Required = true)]
         public string ProjectUriString { get; internal set; }
 
-        // For an explanation of the difference between ProjectUriString and ProjectUri, see FileWorkItemsCommand.ValidateOptions.
+        // ProjectUri is the URI representation of ProjectUriString above. We convert the command-line argument to the URI
+        // form as part of argument validation at runtime (in order to provide better error messages than those that 
+        // the CommandlineLibrary component provides). See FileWorkItemsCommand.ValidateOptions for more details.
         public Uri ProjectUri { get; internal set; }
 
-        [Option(
-            "personal-access-token",
-            HelpText = "TEMPORARY: Specifes the personal access used to access the project")]
-        public string PersonalAccessToken { get; internal set; }
```

Comment:
>PersonalAccessToken [](start = 22, length = 19)

We should probably somehow ensure users are aware of setting environment variable now that this is removed. Maybe document this the verb <code>file-work-items</file>?

Reply:
TODO for next PR. Let's discuss offline.

---
In reply to: [375966103](https://github.com/microsoft/sarif-sdk/pull/1744#discussion_r375966103) [](ancestors = 375966103)